x11-themes/terminology-themes/terminology-themes-9999.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit git-r3
+
+DESCRIPTION="Color schemes for the Terminology terminal emulator"
+HOMEPAGE="https://charlesmilette.net/terminology-themes/"
+SRC_URI=""
+
+EGIT_REPO_URI="https://github.com/sylveon/terminology-themes"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS=""
+IUSE=""
+
+DEPEND="dev-libs/efl"
+RDEPEND="
+	${DEPEND}
```

Comment:
Do you actually need efl at runtime? Not saying it won't be pulled in by other packages but does this one strictly need it?

Reply:
Nope, that is just a build dependency - thanks! :)